components/studio/test/shared/studio-enter.ps1

```diff
@@ -0,0 +1,34 @@
+# Depends on https://www.powershellgallery.com/packages/Await/0.8
+param([String]$studio_command) #
+
+$env:HAB_NOCOLORING = "true"
+# Await has trouble parsing non-ascii glyphs
+$env:HAB_GLYPH_STYLE = "ascii"  
+$exit_code = 0
+
+Install-Module -Name Await
+
+try { 
+    Start-AwaitSession 
+    Send-AwaitCommand "$studio_command enter" 
+    $prompt = Wait-AwaitResponse "[HAB-STUDIO] Habitat:\src>"
+    Write-Host $prompt
+
+    Send-AwaitCommand "test/shared/studio-internals/test-all.ps1"
+    Write-Host "Send test comand"
+     $result = Wait-AwaitResponse "All tests finished"
+
+    Write-Host $result
+    if($result -match "[FAIL]") {
```

Comment:
where is `[FAIL]` emitted?

Reply:
`[FAIL]` should be emitted by each individual test. I'm doing some small refactoring and I'll double check that that is true.   I've also added a README (not pushed yet) explaining how these tests work and what they should do.

Reply:
yeah I see that in the bash/exp but don't see anything on the powershell side that would output `[FAIL]`

Reply:
This should be fixed now.